src/code/code_800F7260.c

```diff
@@ -32,9 +33,8 @@ char D_80133390[] = "SEQ H";
 char D_80133398[] = "    L";
 
 // data
-u8 D_801333A0 = 0;
-
-u8 D_801333A4 = 0;
+u8 sSoundRequestHead = 0;
+u8 sSoundRequestTail = 0;
```

Comment:
are you sure these aren't swapped?

`Audio_ProcessSoundRequests` does `sSoundRequestTail++`
and `Audio_PlaySoundGeneral` does `sSoundRequestHead++`

that seems weird but maybe I'm mistaken

Reply:
Question of definitions. My intuition is that the tail lags behind the head, and the head is where it extends forwards, like in Snake. :snake:

Reply:
took me a few minutes but I get how you see it now, then to prevent confusion what about
`sSoundRequestTail` -> `sSoundRequestFirst` (or Start)
`sSoundRequestHead` -> `sSoundRequestNext`

I think head/tail is confusing because traversal starts at the tail which at least for me is counter-intuitive

Reply:
Changed to ReadIndex/WriteIndex. (But I really do like the concise head/tail terminology personally.)